pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -946,6 +946,9 @@ const (
 
 	// TiDBTxnEntrySizeLimit indicates the max size of a entry in membuf.
 	TiDBTxnEntrySizeLimit = "tidb_txn_entry_size_limit"
+
+	// TiDBInfoSchemaCacheSize indicates the size of infoschema meta data which are cached in V2 implementation.
+	TiDBInfoSchemaCacheSize = "tidb_infoschema_cache_size"
```

Comment:
When modify the variable, should we force trigger reload infoschema?

Reply:
Yes, we can, but it seems we need to register callback function in `SetGlobal` to call the function in domain.
How to do that is a bit complex, considering infoschema reload is called eventually anyway, I skip the force trigger part.